google/resource-snippets/cloudkms-v1/alternatives/krm/cloudkms.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kms.cnrm.cloud.google.com/v1beta1
+kind: KMSCryptoKey
+metadata:
+  labels:
+    foo: ${DEPLOYMENT?} # {"$kpt-set":"deployment"}
+  name: ${DEPLOYMENT?}-cryptokey # {"$kpt-set":"deployment-sub"}
+spec:
+  keyRingRef:
+    name: test-keyring
+  purpose: ENCRYPT_DECRYPT
+---
+apiVersion: kms.cnrm.cloud.google.com/v1beta1
+kind: KMSKeyRing
+metadata:
+  name: test-keyring
+spec:
+  location: us-central1
+---
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicy
+metadata:
+  name: iampolicy
+spec:
+  resourceRef:
+    apiVersion: kms.cnrm.cloud.google.com/v1beta1
+    kind: KMSKeyRing
+    name: test-keyring
+  bindings:
+  - role: roles/cloudkms.admin
+    members:
+    - serviceAccount:zyansh-dm@zyanshu-dm-convert.iam.gserviceaccount.com
```

Comment:
Ideally DM and KRM and TF should all have the same resource data, because we're trying to show different ways of achieving the same result. So how about making the user a setter as well to reflect [DM](https://github.com/GoogleCloudPlatform/deploymentmanager-samples/blob/master/google/resource-snippets/cloudkms-v1/kms.jinja#L34) and [TF](https://github.com/GoogleCloudPlatform/deploymentmanager-samples/blob/master/google/resource-snippets/cloudkms-v1/alternatives/tf/cloudkms.tf#L3)?

Then something like `kpt cfg set . keyring-member ananke.iam@gmail.com`.

I understand using your own accounts for testing (that's what I'd do as well) but when you're checking-in to GitHub, let's avoid hardcoding your personal service accounts.